.github/workflows/rust-wasmedge-types-docs.yml

```diff
@@ -8,7 +8,7 @@ on:
   push:
     branches:
       - "master"
-      - "rust/*"
+      - "release/rust/*"
```

Comment:
Do you want to check the release tag? If so, you should use `release/rust-types/*`.

Reply:
Accepted. Thanks!